src/estimation.jl

```diff
@@ -38,4 +40,166 @@ function esprit(x::AbstractArray, M::Integer, p::Integer, Fs::Real=1.0)
     angle.(D)*Fs/2π
 end
 
+"""
+    jacobsen(x::Vector, Fs::Real = 1.0)
+
+Estimate the largest frequency (in Hz) in the signal `x` using Jacobsen's
+algorithm [^Jacobsen2007].
+
+[^Jacobsen2007]: E Jacobsen and P Kootsookos, "Fast, Accurate Frequency Estimators", Chapter
+10 in "Streamlining Digital Signal Processing", edited by R. Lyons, 2007, IEEE Press.
+"""
+function jacobsen(x::Vector{<:Real}, Fs::Real = 1.0)
+    N = length(x)
+    X = rfft(x)
+    k = argmax(abs.(X))  # index of DFT peak
+    if (k+1 <= N) && (k-1 >= 1)
+        δ = -real((X[k+1] - X[k-1]) / (2X[k] - X[k-1] - X[k+1]) )
+    else
+        δ = 0.0
+    end
+
+    (k + δ)*Fs/N
```

Comment:
This seems to be off by one:
```suggestion
    return (k - 1 + δ)*Fs/N
```

Reply:
Yes, `k` here is off by one from the DFT definition. I will add some related tests...